Lenovo/L38111/res/values/config.xml

```diff
@@ -0,0 +1,159 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <fraction name="config_autoBrightnessAdjustmentMaxGamma">300.0%</fraction>
+    <fraction name="config_maximumScreenDimRatio">20.000004%</fraction>
+    <string-array name="config_defaultPinnerServiceFiles">
+        <item>/system/framework/arm64/boot-framework.oat</item>
+        <item>/system/framework/arm64/boot-framework.vdex</item>
+        <item>/system/framework/oat/arm64/services.odex</item>
+        <item>/system/framework/oat/arm64/services.vdex</item>
+        <item>/system/framework/arm64/boot.oat</item>
+        <item>/system/framework/arm64/boot.vdex</item>
+        <item>/system/framework/arm64/boot-core-libart.oat</item>
+        <item>/system/framework/arm64/boot-core-libart.vdex</item>
+    </string-array>
+    <integer-array name="config_autoBrightnessLevels">
+         <item>3</item>
+        <item>41</item>
+        <item>82</item>
+        <item>124</item>
+        <item>206</item>
+        <item>275</item>
+        <item>481</item>
+        <item>618</item>
+        <item>756</item>
+        <item>989</item>
+        <item>1305</item>
+        <item>1511</item>
+        <item>2061</item>
+        <item>2748</item>
+        <item>3100</item>
```

Comment:
Auto brightness really works with this?

Reply:
Yes, it does, actually. All the configs have been taken from the official rom.

Reply:
Official ROM can have brightness go up to 3100, but AOSP's max is 255

Reply:
I see, I will change the brightness levels.